Changelog.md

```diff
@@ -9,12 +20,12 @@ Compiler Features:
 
 
 Bugfixes:
```

Comment:
This is just a cherry-pick from the other PR so we don't have to wait until its merged.